### DISCUSSION TITLE
Is it possible to eager load more related models?

### DISCUSSION THREAD

**Discussion by isgj** [SeaQL/sea-orm#528] [Q&A] [answered]
Hey, if we have model like
```
Model {
    id: ...
    rel_1_id: ...
    rel_2_id: ...
}
```
is it possible to load the main model and also the related ones?
```
main::Model::find().find_also_related(Rel_1).find_also_related(Rel_2)...
```

**Comment by billy1624** [comment, marked as answer]
Hey @isgj, currently only support single related models, https://www.sea-ql.org/SeaORM/docs/basic-crud/select#find-related-models.

We can discuss it on related issue
- #440

**Reply by isgj** [reply]
The linked issue is different, it's about deeper relations

```mermaid
classDiagram
      Foo <-- Bar
      Bar <-- Qux
```

this is about 1 level deep relations

```mermaid
classDiagram
  Foo <-- Bar
  Foo <-- Qux
```

but yeah, they can be discussed  together.

**Reply by billy1624** [reply]
Actually, this is also related. And it address the multiple related entities on the same depth that you said
- #415
